src/averageAges.js

```diff
@@ -14,12 +14,17 @@
  *
  * @return {number}
  */
+const sumAgeDifference = (sum, x) => sum + (x.died - x.born);
+const averageAge
+= (peopleArray) => peopleArray.reduce(sumAgeDifference, 0) / peopleArray.length;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = century
+    ? people.filter((x) =>
+      ((x.sex === 'm') && (century === Math.ceil(x.died / 100))))
+    : people.filter((x) => ((x.sex === 'm')));
```

Comment:
In both cases you filter people. You can move ternary operator inside filter callback to make the code a bit cleaner